docs/source/AdministratorGuide/Systems/DataManagement/concepts.rst

```diff
@@ -0,0 +1,37 @@
+--------
+Concepts
+--------
+
+The whole DataManagement System (DMS) of DIRAC relies on a few key concepts:
+
+* Logical File Name (LFN): the LFN is the name of a file, a path. It uniquely identifies a File throughout the DIRAC namespace. A file can have one or several `Replica`.
+* Replica: This is a physical copy of an LFN. It is stored at a `StorageElement`. The couple `(LFN,StorageElement)` uniquely identifies a physical copy of a file.
+* StorageElement: This represents a physical storage endpoint.
+* Catalog: This is the namespace of the DataManagement. Files and their metadata are listed there
+
+Systems in DIRAC (other than DMS) or users, when dealing with files, only have to care about LFNs. If, for some (unlikely) reasons, they need to address a specific replica, then they should use the couple `(LFN, StorageElement name)`. At no point, anywhere, is there a protocol or a URL leaking out of the low level of the DMS.
+
```

Comment:
is there -> there is

Reply:
Hum, not sure.  For `At no point in time`, what I wrote would be correct (http://www.dictionary.com/browse/point-in-time).